src/BuiltInTools/dotnet-watch/dotnet-watch.csproj

```diff
@@ -33,11 +33,6 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" />
     <PackageReference Include="Microsoft.NET.StringTools" ExcludeAssets="runtime" PrivateAssets="All" />
 
-    <!--
-      Explicit System.Text.Json package reference is required for source-build to pick up the live package.
-      Avoids picking up an old version via transitive dependency from Microsoft.Build or Microsoft.CodeAnalysis.Workspaces.MSBuild.
-    -->
-    <PackageReference Include="System.Text.Json" />
```

Comment:
Probably want a review from @tmat but with CPM now enabled, this may not be required anymore like it was before.

Reply:
Having an explicit package reference is no longer required, or removing it is no longer required?

Reply:
Having the explicit reference is no longer required, I think. If I understand the original issue per the comment, a transitive version was picked up that didn't work with source build and adding the explicit reference up-versioned to one that was supported. CPM should do the version change as well.